test/init_tracer.js

```diff
@@ -156,4 +155,23 @@ describe('initTracer', () => {
         assert.equal(tracer._metrics._factory, metrics);
         assert.equal(tracer._tags['x'], 'y');
     });
+
+    describe("upsampling", () => {
+        let config = {
```

Comment:
Why not use `const`? Side note: Eslint can be config'd to detect when declarations should use `const`.